CodeiumVS/CodeiumVS.csproj

```diff
@@ -126,8 +126,8 @@
     <PackageReference Include="Community.VisualStudio.Toolkit.17" Version="17.0.507" ExcludeAssets="Runtime">
       <IncludeAssets>compile; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.VisualStudio.Language">
-      <Version>17.8.222</Version>
```

Comment:
Removed the `Microsoft.VisualStudio.Language` package because it's already included in `Microsoft.VisualStudio.SDK`